### ISSUE TITLE
Button html is a bit strange

### ISSUE THREAD

**Issue by developess** [fac-14/HMEN#33]
https://github.com/fac-14/HMEN/blob/3bb71dfc878205f52110bfe09f5e5c13754aa349/public/index.html#L20

Typically rather than having a button as an input and setting text with a value, people use a button html tag:

```
<button type=submit>Get News</button>
```
Then you won't need the label for the button.

You can set an event listener on the **form** for a "submit" event - which will work for button clicks and enter key presses - so you only need one.

**Comment by sima-qian** [comment]
Yeah, I think I had problems in the past with not being able to stop a form submit from automatically reloading the page, so we went with `type=button` over `type=submit` but I guess that was unnecessary -- thanks for the advice 😍
